packages/venia-ui/lib/components/Gallery/__tests__/__snapshots__/item.spec.js.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders a placeholder item while awaiting item 1`] = `
+<div
+  className="l"
+>
+  <div
+    className="f"
+  >
+    <img
+      alt=""
```

Comment:
Is there a good `alt` value we can use for a placeholder?

Reply:
It is a temporary placeholder but the `alt` of the image is the `item.name`, and this only gets rendered when `item` doesn't exist (yet).

I went with "Placeholder for gallery item image" 👍 .

Reply:
https://github.com/magento/pwa-studio/pull/1871/commits/70dc5fd9cc7db80d41403353df8037299713898b